PSEUDO API 1.md

```diff
@@ -0,0 +1,272 @@
+
+## XR
+
+	interface XR {
+		Promise<sequence<XRDisplay>> getDisplays();
+
+		attribute EventHandler ondisplayconnect;
+		attribute EventHandler ondisplaydisconnect;
+	};
+
+_The interfaces with "VR" in the name have been changed to "XR" to indicate that they are used for both VR and AR._
```

Comment:
Commentary that indicates changes from the WebVR "2.0" working spec are denoted with italics.